README.md

```diff
@@ -182,6 +183,16 @@ To start developing, please refer to [CONTRIBUTING.md][contributing].
 
 [contributing]: https://github.com/cameleon-rs/cameleon/blob/main/CONTRIBUTING.md
 
+## Releasing
+### 1. Publish
+1. Bump up all the crate versions in the workspace using `cargo release version <major|minor|patch>`
+2. Open a PR to reflect the changes.
+3. Publish the crates with [`cargo release`](https://github.com/crate-ci/cargo-release)
```

Comment:
This could be squashed to just 2 steps
1. `cargo release <major|minor|patch>`
2. Open PR

Tt felt more defensive to split it a bit more, but tell me what you think.

Reply:
I feel it'd be slightly better to make the description concise, assuming the number of maintainers is not so large. But, I don't have a strong feeling about this, so please feel free to merge this PR.

Reply:
In that case let me make it a bit more concise. But wait, let me also document an important step that should be done first.
```suggestion
1. Check commits since the last release and determine whether they're semver-breaking.
2. Bump up all the crate versions and publish using [`cargo release <major|minor|patch>`](https://github.com/crate-ci/cargo-release)
3. Open a PR to reflect the changes.
```
@Y-Nak please check accept the suggestion right away if it feels good.